python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```
@@ -422,13 +422,19 @@ $defs:
                     type: str
                     description: |
                       IPv4_address/Mask.
-                      Loopback ip range, a unique ip is derived from this ranged and assignedto each l3 leaf based on it's unique id.
+                      Loopback ip range, a unique ip is derived from this range and assigned to each l3 leaf based on it's unique id.
                       Loopback is not created unless loopback_ip_range or loopback_ip_pools are set.
+                  loopback_ipv6_range:
+                    type: str
+                    description: |
+                      IPv6_address/Mask.
+                      Loopback ipv6 range, a unique ipv6 is derived from this range and assigned to each l3 leaf based on it's unique id.
+                      Loopback is not created unless loopback_ipv6_range or loopback_ip_pools are set.
```

This statement seems wrong now.
```suggestion
                      Loopback is not created unless `loopback_ip_range`, `loopback_ipv6_range` or `loopback_ip_pools` are set.
```